test/markdown-link-check.test.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+/* jscpd:ignore-start */
```

Comment:
What is this for?

Reply:
[jscpd](https://github.com/kucherenko/jscpd/tree/master/packages/jscpd#readme) is a copy-paste detector (because too big copy-pastes are evil )
It detected one in this file, but I considered it minor so added comments so jscpd ignores it

Reply:
I can also disable copy-paste detector if you like